modules/modal/css/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@workday/canvas-kit-css-modal",
+  "version": "3.0.0-alpha.6",
+  "description": "Modal CSS for Canvas kit CSS",
+  "author": "Workday, Inc. (https://www.workday.com)",
+  "license": "Apache-2.0",
+  "files": [
+    "dist",
+    "lib",
+    "index.scss"
+  ],
+  "style": "dist/canvas-kit-css-modal.min.css",
+  "main": "dist/canvas-kit-css-modal.min.css",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/Workday/canvas-kit/tree/master/modules/modal/css"
+  },
+  "dependencies": {
+    "@workday/canvas-kit-css-core": "^3.0.0-alpha.6"
```

Comment:
Missing deps here

Reply:
I take it we need a `"@workday/canvas-kit-css-popup": "^3.0.0-alpha.6"` here? Does the CSS Popup `package.json` need `"@workday/canvas-kit-css-card": "^3.0.0-alpha.6"` as well then?

Is there an error or process that flags missing dependencies or is this something we need to be mindful of doing ourselves?

Reply:
Yep exactly. You are correct - good catch! Do you mind adding it?

It's in our roadmap to add CI to check for missing deps, but we don't have it yet so we need to be mindful (for now 😅)

Reply:
Sure thing. On a related note, does the CSS Popup `index.scss` need to import CSS Button?

https://github.com/Workday/canvas-kit/blob/37160e03a87948cea865f08abf5a13272a262669/modules/popup/css/index.scss#L3

The two are often used together, but unless I'm missing something I don't think Popup needs the Button styles.

Reply:
I updated the dependencies in both `package.json`s. I also removed the CSS Button import from the CSS Popup `index.scss`.